cinema/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255)
+    description = serializers.CharField(max_length=255)
+    duration = serializers.IntegerField()
+
+    def create(self, validated_data):
+        return Movie.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.title = validated_data.get("name", instance.name)
```

Comment:
There is a mistake in the `update` method: `validated_data.get("name", instance.name)` should be `validated_data.get("title", instance.title)`. The field name should match the model's field name, which is `title`.